workshops/templatetags/diff.py

```diff
@@ -9,3 +10,52 @@
 @register.simple_tag
 def semantic_diff(left, right, field):
     return mark_safe(generate_patch_html(left, right, field, cleanup='semantic'))
+
+
+@register.simple_tag
+def relation_diff(left, right, field):
+    if field.many_to_one or field.one_to_one:
+        # left.field_dict[field.name] is an integer or does not exist
+        # Cast it to a list(empty or single itemed)
+        if left.field_dict.get(field.name):
+            left_PKs = [left.field_dict.get(field.name)]
+        else:
+            left_PKs = []
+        if right.field_dict.get(field.name):
+            right_PKs = [right.field_dict.get(field.name)]
+        else:
+            right_PKs = []
+    else:
+        left_PKs = left.field_dict.get(field.name,[])
+        right_PKs = right.field_dict.get(field.name,[])
+    # Relations that exist only in the current version
+    additions = field.related_model.objects.filter(
```

Comment:
Bug: Does not show objects that have been deleted.
I'm working on a fix for this.